frontend/app/src/filters.ts

```diff
@@ -32,6 +32,19 @@ export function roundDown(value: BigNumber, precision: number) {
   return value.toFormat(precision, BigNumber.ROUND_DOWN);
 }
 
+// truncates blockchain addresses retaining `truncLength` characters
```

Comment:
instead of a comment can you at least make it proper jsdoc

Reply:
ok